tuna/corrupt_configs.py

```diff
@@ -47,31 +47,50 @@ def get_logger():
   return logger
 
 
-def main():
+LOGGER = get_logger()
+
+
+# pylint: disable-msg=too-many-locals
+def main() -> None:
   """Main module function"""
-  file_name = None
-  arch = None
+  file_name: Optional[str] = None
+  arch: Optional[str] = None
+
   if len(sys.argv) == 3:
     file_name = sys.argv[1]
     arch = sys.argv[2]
   else:
     sys.exit("Usage: " + sys.argv[0] + " <output_file> <arch>")
 
-  table_cols_conv_invmap = {v: k for k, v in TABLE_COLS_CONV_MAP.items()}
-  table_cols_fusion_invmap = {v: k for k, v in TABLE_COLS_FUSION_MAP.items()}
+  idx_1: Dict[Any, Any]
+  idx_2: Dict[Any, Any]
+  table_cols_conv_invmap: dict = {
+      idx_1: idx_2 for idx_2, idx_1 in TABLE_COLS_CONV_MAP.items()
+  }
+  table_cols_fusion_invmap: dict = {
+      idx_1: idx_2 for idx_2, idx_1 in TABLE_COLS_FUSION_MAP.items()
+  }
 
+  outfile: TextIOWrapper
+  cur: Any
   with open(file_name, "w") as outfile:  # pylint: disable=unspecified-encoding
-    sub_cmd_idx = None
+    sub_cmd_idx: Union[Any, slice] = None
     with DbCursor() as cur:
       cur.execute(
-          "select config.* from job inner join config on config.id = job.config where \
-              job.arch = %s and reason = 'corrupt';", (arch,))
+          "select conv_config.* from job inner join conv_config on conv_config.id = \
```

Comment:
There is no job table and conv_job doesn't have an arch column.

Reply:
For this change, I am going to raise a saparate PR. In that , this querry would be re written as per the present table structures.